src/main/java/com/codeus/winter/config/DefaultBeanFactory.java

```diff
@@ -77,38 +110,56 @@ public Object createBean(Class<Object> beanClass)
         return newBean;
     }
 
+    /**
+     * Creating bean object with name.
+     * @param name bean's name
+     * @return bean object if its not possible throw exception.
+     */
     @Override
-    public Object createBean(String name) throws NotUniqueBeanDefinitionException {
+    public Object createBean(@Nonnull final String name) throws NotUniqueBeanDefinitionException {
         uniqueBeanChecking(name);
```

Comment:
I would suggest renaming this method to smth like checkBeanNameUniqueness(), so it would be a bit clearer that we're going to check a bean name and not a bean itself, and, of course, if you have any better naming options I'd be glad to see them;)